jsettlers.tools/gradle.properties

```diff
@@ -0,0 +1,14 @@
+netbeans.org-netbeans-modules-editor-indent.text.x-java.CodeStyle.project.text-line-wrap=none
```

Comment:
If I am not mistaken no IDE except netbeans will try to interpret these lines. I don't want any IDE specific stuff in the git repo so please remove this

Reply:
So far you eventually point out that tabs have to be used, or that the license header is not correct. I find it rather annoying to see such primitive things holding up either a quick merge or a discussion on the purpose of  a PR.

So the point is to agree and enforce on code quality, and I (and others) can directly get this feedback on every build rather than just before the merge. it will remove frustration on both sides.

The fact that it generates a lot of warnings shows that such code style was never enforced, so one could question why you are asking it in first place. But we are not just talking about formatting with 4 spaces or 8 or tabs. Checkstyle can check for so much more that makes the code easier readable, understandable, maintainable. 

So even if we do not agree on the concise configuration it currently has: let's activate some checks that we are sure to apply, and do others later if we feel like.

In short: Do not question the pull request. Come up with rules you feel comfortable about and activate them.